src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/CreateInductionTest.kt

```diff
@@ -263,4 +266,72 @@ class CreateInductionTest : IntegrationTestBase() {
           .hasWorkedBeforeNotRelevantReason("Prisoner is not looking for work so feels previous work experience is not relevant")
       }
   }
+
+  @Test
```

Comment:
New tests to create an induction before the action plan and assert no review schedule created; and a test to create the induction after then action plan and to assert the review schedule is created.